TestPlaybooks/playbook-Process_Email_-_Generic_-_Test.yml

```diff
@@ -304,7 +304,7 @@ tasks:
         simple: "yes"
       unsecure: {}
       url:
-        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docwithindicators.doc
+        simple: https://raw.githubusercontent.com/demisto/content/master/TestData/docwithindicators
```

Comment:
Why did you remove the `.doc`. Seems like a bug

Reply:
this was done by mistake (refactored another item by this name, and this happened automatically). Fixing it